samanthas_telegram_bot/conversation/user_data.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+from typing import Literal
+
+from samanthas_telegram_bot.conversation.constants_enums import Role
+
+
+@dataclass
+class TeacherPeerHelp:
+    """A class that comprises boolean flags for experienced teachers' willingness to help their
+    peers.
+    """
+
+    can_consult_other_teachers: bool = None
+    can_help_with_children_group: bool = None
+    can_help_with_materials: bool = None
+    can_check_syllabus: bool = None
+    can_give_feedback: bool = None
+    can_invite_to_class: bool = None
+    can_work_in_tandem: bool = None
+
+
+@dataclass
+class UserData:
+    # TODO sets instead of lists?
+    locale: str = None
```

Comment:
`locale: str = None` has wrong typing. It should be `locale: str | None` if it could be None. Here and below

Reply:
What I meant was `locale: str | None = None`. I saw this in `python-telegram-bot`, it's everywhere like this in there, although the guys are really into good type hints. So I thought I could use that shorthand too :)

Reply:
What mypy thinks about it? 🤔 

You definitely could understand by this code that this field is optional, but in context of typing it's typed wrongly :)

Reply:
OK, I see now, the guys use [`implicit-optional`](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-implicit-optional) for arguments, this won't work here in the dataclass anyway.

Reply:
Fixed mypy issues